### PR TITLE
Updating the README to remove references to "global" key

### PR DESCRIPTION
**Pull request by adambollen**
Fauna doesn't support the creation of a "global" database key for interacting with all databases at once, so this change updates the README file to account for that.  The change also includes a few wording tweaks for clarity, including links to the Region Group documentation to make sure users aren't tripped up by getting 401s when pointing at the wrong endpoint.

### CODE REVIEW

**Review by cleve-fauna — COMMENTED** [review]
Some suggestions on local dev guidance.

**Review by cleve-fauna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -51,22 +53,22 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
 
 ![Extension settings](media/extension-settings.png)
 
-- `fauna.adminSecretKey`: The secret for a specific database.
-- `fauna.domain`: The hostname of endpoint’s Fauna instance
-- `fauna.scheme`: One of https or http
-- `fauna.port`: The UNIX port number of endpoint’s Fauna instance
-- `fauna.graphqlHost`: The hostname of Fauna GraphQL API
+- `fauna.adminSecretKey`: The secret for a specific database
+- `fauna.domain`: The Fauna domain for the database's [region group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance
```
Comment (faunaee) [inline README.md:57]:
```suggestion
- `fauna.domain`: The Fauna domain for the database's [Region Group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance.
```

File: README.md [hunk 2]
```diff
@@ -51,22 +53,22 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
...
+- `fauna.scheme`: One of `https` or `http` (only set to `http` when using a local Fauna Dev Docker instance)
```
Comment (faunaee) [inline README.md:58]:
```suggestion
- `fauna.scheme`: One of `https` or `http`. Only set to `http` when using a local Fauna Dev Docker instance).
```

File: README.md [hunk 3]
```diff
@@ -51,22 +53,22 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
...
+- `fauna.port`: The port number for the Fauna endpoint (when using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`)
```
Comment (faunaee) [inline README.md:59]:
```suggestion
- `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`.
```

File: README.md [hunk 4]
```diff
@@ -51,22 +53,22 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
...
+- `fauna.graphqlHost`: The full URL for the Fauna GraphQL API (e.g. `https://graphql.fauna.com`)
```
Comment (faunaee) [inline README.md:60]:
```suggestion
- `fauna.graphqlHost`: The full URL for the Fauna GraphQL API. The default is `https://graphql.fauna.com`. See [GraphQL Endpoints](https://docs.fauna.com/fauna/current/api/graphql/endpoints) for Region Group-specific URLs.
```

File: README.md [hunk 5]
```diff
@@ -51,22 +53,22 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
 
 ![Extension settings](media/extension-settings.png)
 
-- `fauna.adminSecretKey`: The secret for a specific database.
-- `fauna.domain`: The hostname of endpoint’s Fauna instance
-- `fauna.scheme`: One of https or http
-- `fauna.port`: The UNIX port number of endpoint’s Fauna instance
-- `fauna.graphqlHost`: The hostname of Fauna GraphQL API
+- `fauna.adminSecretKey`: The secret for a specific database
```
Comment (faunaee) [inline README.md:56]:
```suggestion
- `fauna.adminSecretKey`: The secret for a specific database.
```

Add a period to the end of each of the preceding bullet points.

**Review by faunaee — DISMISSED** [review]
Just a few minor improvements.

**Review by cleve-fauna — APPROVED** [review]

**Comment by adambollen** [comment]
I updated the Settings to match the README; should look like this now:
image

**Comment by adambollen** [comment]
New Settings descriptions:
image

**Review by cleve-fauna — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -52,21 +54,21 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
 ![Extension settings](media/extension-settings.png)
 
 - `fauna.adminSecretKey`: The secret for a specific database.
-- `fauna.domain`: The hostname of endpoint’s Fauna instance
-- `fauna.scheme`: One of https or http
-- `fauna.port`: The UNIX port number of endpoint’s Fauna instance
-- `fauna.graphqlHost`: The hostname of Fauna GraphQL API
+- `fauna.domain`: The Fauna domain for the database's [Region Group](https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups) or `localhost` when using a local Fauna Dev Docker instance.
+- `fauna.scheme`: One of `https` or `http`. Only set to `http` when using a local Fauna Dev Docker instance.
+- `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`.
```
Comment (faunaee) [inline README.md:59]:
```suggestion
- `fauna.port`: The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, use the port passed to the `docker run` command - usually `8443`.
```

File: README.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 Source code for the [Fauna VS Code extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna).
 
-This [extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna) allows users to browse their Fauna databases and run [FQL queries](https://docs.fauna.com/fauna/current/api/fql/) from directly inside VS Code.
+This [extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna) allows users to view individual Fauna databases and run [FQL queries](https://docs.fauna.com/fauna/current/api/fql/) from directly inside VS Code.
```
Comment (faunaee) [inline README.md:5]:
```suggestion
This [extension](https://marketplace.visualstudio.com/items?itemName=fauna.fauna) allows users to view individual Fauna databases and run [FQL queries](https://docs.fauna.com/fauna/current/api/fql/) directly inside VS Code.
```

File: README.md [hunk 8]
```diff
@@ -52,21 +54,21 @@ FAUNA_GRAPHQL_HOST=graphql.fauna.com
...
+- `fauna.graphqlHost`: The full URL for the Fauna GraphQL API. The default is `https://graphql.fauna.com`. See [GraphQL Endpoints](https://docs.fauna.com/fauna/current/api/graphql/endpoints) for Region Group-specific URLs.
 
 > WARNING: Be careful! To avoid exposing this secret, do not commit it to your local `.vscode` configuration.
 
-### 2. Browse database/s
+### 2. Browse database
 
 1. Click on the Fauna bird icon in the Activity bar on the far left. If you do not see the Activity Bar, select `View > Appearance > Show Activity Bar` from the VS Code menu.
-2. You should now see a pane listing all of your databases, indexes, collections, documents, and functions.
+2. You should now see a pane listing all of your database's contents, including child databases, indexes, collections, documents, and functions.
```
Comment (faunaee) [inline README.md:67]:
```suggestion
2. You should now see a pane listing your database's contents, including child databases, indexes, collections, documents, and functions.
```

File: package.json [hunk 9]
```diff
@@ -48,17 +48,17 @@
...
             "http"
           ],
-          "description": "Optional - One of https or http. Defaults to https"
+          "markdownDescription": "Optional - One of `https` or `http`. Only set to `http` when using a local Fauna Dev Docker instance."
         },
         "fauna.port": {
           "type": "number",
           "default": 443,
-          "description": "Optional - The UNIX port number of endpoint’s Fauna instance. Defaults to 443"
+          "markdownDescription": "Optional - The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, be sure to use the port passed to the `docker run` command - usually `8443`."
```
Comment (faunaee) [inline package.json:56]:
```suggestion
          "markdownDescription": "Optional - The port number for the Fauna endpoint. When using a Fauna Dev Docker instance, use the port passed to the `docker run` command - usually `8443`."
```

**Review by faunaee — APPROVED** [review]
A few final word-smithing suggestions. Otherwise, looks great!

**Review by cleve-fauna — DISMISSED** [review]

**Review by faunaee — DISMISSED** [review]
LGTM! 👍

**Review by ghartsel — DISMISSED** [review]
Change as you feel appropriate.

**Review by cleve-fauna — APPROVED** [review]

**Pull request merged**